types/statsd-client/tsconfig.json

```diff
@@ -5,6 +5,7 @@
             "es6",
             "dom"
         ],
+        "esModuleInterop": true,
```

Comment:
AFAIK your package does not require this, that's standard CommonJS module:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44595#discussion_r425400276
https://github.com/msiebuhr/node-statsd-client/blob/master/lib/statsd-client.js#L186
That's for end-users (consumers of the DT module), not  intended to be used here for tests.
So your tests would have to be corrected IMO:
```ts
import SDC = require('statsd-client');
import { Tags, TcpOptions, UdpOptions, HttpOptions } from 'statsd-client';
```